llvm/unittests/ProfileData/InstrProfTest.cpp

```diff
@@ -581,6 +581,52 @@ TEST_F(InstrProfTest, test_memprof_v2_partial_schema) {
   EXPECT_THAT(WantRecord, EqualsRecord(Record));
 }
 
+TEST_F(InstrProfTest, test_memprof_missing_callstackid) {
```

Comment:
I don't think this needs anything from the fixture? If so can we move it to MemProfTest.cpp since it's exclusive to MemProf?

Reply:
Good point.  I just moved them in the latest iteration.  Please take a look.  Thanks!